### DISCUSSION TITLE
What is the state of the art for HTML in scholarly publishing in 2022?

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#36] [Q&A]
We are looking to see what are the current flavors of HTML in use for scholarly publishing.

At various times different projects have sprung up with recommendation for good practice in the use of HTML for scholarly publishing, whether as an output format, or as a interchangeable format. But in 2022 it's a bit hard to know what projects are out there and what are their merits.

If people have any suggestions of projects to take a look at, of good practice, how HTML can be used, HTML weaknesses/strengths - input appreciated.

We'll  keep a record of notes on this open pad and please feel free to post here in the discussion.

https://demo.hedgedoc.org/AK4qMLIDRoyAPb31QsNXvQ

**Comment by hauschke** [comment]
https://w3c.github.io/scholarly-html/ is AFAIK one of the more mature approaches. But it didn't receive any discussion in the last few years, so very likely it didn't receive a lot of uptake.

**Reply by johanneswilm** [reply]
Hey @hauschke that was also where I looked first. But clicking from there to the email archives of that group and the last conversation that took place in 2020 [1], it sounded like everyone has left and participants have found other alternatives.

[1] https://lists.w3.org/Archives/Public/public-scholarlyhtml/2020Mar/0000.html

**Comment by hubgit** [comment]
I think in general there’s not much about “scholarly” HTML that makes it different from any other kind of article, so apart from the `citation_*` meta tags that Google Scholar likes to see in the `head` of the article, the usual general principles of publishing content online apply:


- Can machines understand the content?
	- Is the metadata available as JSON-LD or HTML5 microdata?
	- Are the alternate formats (e.g. PDF), versions,  commentary/reviews and license discoverable using link relations?
- Can people understand the content?
	- Is it accessible to everyone?
	- It is fast to load, easy to read and not surrounded by junk?
	- Are the alternate formats (e.g. PDF), version, commentary/reviews and license easily discoverable?

Here’s [an example of an HTML article that I think is pretty good](https://peerj.com/articles/12638.html). It’s using schema.org for the metadata; the one thing still missing is an agreed-upon way of defining all the pieces that make up the whole article, particularly the supplementary files.

**Reply by johanneswilm** [reply]
Hey @hubgit Schema.org does sound good. But if added to an HTML output filter, will it be enough to ensure that other tools will be able to reuse the sources? Preferably, I'd want to follow some kind of authoritative source as closely as possible for all the elements.

> It’s using schema.org for the metadata; the one thing still missing is an agreed-upon way of defining all the pieces that make up the whole article, particularly the supplementary files.

Is any work being done on this anywhere?

**Comment by rowanc1** [comment]
Maybe [Curvenote](https://curvenote.com) might interest you? Curvenote has HTML-first editing of scientific content (articles/papers/textbooks). Including lots of transformations between different formats (pdf, latex, word, jupyter notebooks), and keeping all of the various components versioned and in-sync (single-source CMS).

**Reply by johanneswilm** [reply]
@rowanc1 Is there a particular subtype of HTML that they promote? Is their format standardized?
